### ISSUE TITLE
Addition of noise to break symmetry

### ISSUE THREAD

**Issue by VinF** [soumith/net2net.torch#3]
Hi, 
When you replicate the randomly chosen units, it seems that you don't add noise to break symmetry after the replication? If so, I think this may lead to not actually increasing the network capacity.

**Comment by soumith** [comment]
yes, i dont do that. kinda surprising that no one else noticed. However, because of precision limits, symmetry will slowly break..., after an epoch or so.
